tests/models/hf/test_hf_mpt_gen.py

```diff
@@ -24,6 +24,12 @@ def test_init_hfhub_mpt(
         pytest.skip(f'{attn_impl=} not implemented for {device=}.')
     composer_device = get_device(device)
 
+    if device == 'gpu' and attn_impl == 'flash':
```

Comment:
error should be in the code, not in the test. test can confirm that the error is raise